pkgs/by-name/gp/gpustat/package.nix

```
@@ -0,0 +1,62 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, fontconfig
+, libGL
+, xorg
+, cmake
+, libxkbcommon
+, wayland
+, makeWrapper
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "gpustat";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "arduano";
+    repo = pname;
```

```suggestion
    repo = "gpustat";
```

backrefrencing pname is discouraged, see #277994